pyproject.toml

```diff
@@ -115,6 +115,7 @@ open_files_ignore = ['test.fits', 'asdf.fits']
 filterwarnings = [
     'error',
     'ignore:numpy.ndarray size changed:RuntimeWarning',
+    'ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.entry_points:27',
```

Comment:
```suggestion
```
This is unnecessary, I just ran these changes without this and the tests passed just fine.